src/assessment.js

```diff
@@ -47,7 +47,11 @@ var q2 = [/* your answer here */];
 // context of `alderaan`. In other words, we want to *reference* the `fire`
 // method and temporarily bind it (or, change `this`) to `alderaan` before
 // executing it.
-//
+
+
 var q5 = function fireOnAlderaan () {
-  // your code here
-};
+  console.log(this);
+}
+fireOnAlderaan.call(alderaan);
+
+This is the Call/Apply Invocation Pattern. Not sure how to fully execute it.
```

Comment:
You're correct - that's the pattern to use.  In this question, we want `fireOnAlderaan` to tell the `deathStar` to `fire` on `alderaan`. Therefore, we want to invoke `deathStar.fire` in the context (`this`) of `alderaan`.